shuup/campaigns/admin_module/forms/_basket_effects.py

```diff
@@ -40,3 +40,21 @@ class Meta(BaseEffectModelForm.Meta):
 class DiscountFromProductForm(BaseEffectModelForm):
     class Meta(BaseEffectModelForm.Meta):
         model = DiscountFromProduct
+
+
+class DiscountFromCategoryProductsForm(BaseEffectModelForm):
+    discount_percentage = PercentageField(
+        max_digits=6, decimal_places=5,
+        label=_("discount percentage"),
+        help_text=_("The discount percentage for this campaign."))
+
```

Comment:
No it's not missing it is a model form and the discount_percentage is just there to ovverride the fornfield to `PercentageField`